python/drned_xmnr/op/config_op.py

```diff
@@ -195,32 +195,24 @@ def import_file(self, source_file, state):
                         outfile.write(line)
             if self.merge:
```

Comment:
`self.merge` - is it still needed if merge is removed? Same line 213.

Reply:
Yes, it is. The "merge" option still exists and if set, it needs to be reflected; only the behavior had to be changed for the case when it is not set (and only for XML imports).